web/static/apps/wos/wos.js

```diff
@@ -0,0 +1,38 @@
+// TODO not sure about how to get the server_address properly
+const server_address = "http://localhost:8081";
+
+async function display_wall() {
+    let response = await fetch(server_address + "/wos/users");
+    let content = document.getElementById("content");
+    try {
+        let users_data = await response.json();
+        if (users_data.length == 0)
+            throw "No user in debt";
+        let table = 
+        `<table>
+        <tr>
+          <th>Name</th>
+          <th>Debt($)</th>
+        </tr>` +
+        users_data.map(
+            (users) =>
+                `<tr>
+                <td>${users.username}</td>
+                <td class="balance">${users.balance}</td>
+                </tr>`
+        ).join("")
+        +
+        `</table>`
+        content.innerHTML = table;
+    } catch(e) {
+        content.innerHTML = `<div id="message">No one on the Wall Of Shame
+                            <br>
+                            You should still pay your debts! </div>`;
+    }
+}
+
+// display wall info when loading the page
+display_wall();
+
+// refresh the wall
+setInterval(display_wall, 1000);
```

Comment:
This interval is a little aggressive. We should easily to `60000` (per minute).

Also for important constants like this we should lift them to the top of the file as a const variable like so:

```js
const REFRESH_INTERVAL = 60000; //ms
```